typed-racket-test/unit-tests/contract-tests.rkt

```diff
@@ -226,7 +226,9 @@
    (t-sc -Number number/sc)
    (t-sc -Integer integer/sc)
    (t-sc (-lst Univ) (listof/sc any-wrap/sc))
-   (t-sc (Un (-lst Univ) -Number) (or/sc number/sc (listof/sc any-wrap/sc)))
+   (t-sc (Un (-lst Univ) (-val #t)) (or/sc (flat/sc #''#t) (listof/sc any-wrap/sc)))
```

Comment:
I changed this test because the change to `type-contract.rkt` flattens `number/sc` in the result to all its cases.